test/circuit-breakers.test.js

```diff
@@ -40,34 +39,51 @@ describe('circuit breakers', function(){
 			breaker.numBuckets = 1;
 			breaker.volumeThreshold = 1;
 
+			return app.listen(4242);
 		});
 
-		it('should trip circuit if service fails', function(done){
-			fetch('http://localhost:3000/__fail')
+		afterEach(function() {
+			// HACK: Reset the breaker
+			breaker._buckets = [breaker._createBucket()];
```

Comment:
possibly because of the fact we're running two apps …… there might be multilpe circuit breakery interfering with each other…… might want to move the code that sets up the circuit breaker for this app into the main fixture app so that there's less chance of things interfering with each other……